### ISSUE TITLE
MKS DLC 32 MAX MANUAL PLEASE

### ISSUE THREAD

**Issue by Kostt3d** [makerbase-mks/MKS-DLC32#513]
hello, when will we have the manual please ?

**Comment by Deun67** [comment]
Yes, it would be nice to have the instructions because so far I haven't found anything.

**Comment by alexvtol** [comment]
Please, include a manual how to upload "MKS--DLC32-MAX-main/screen_firmware" too. Thanks in advance

**Comment by Mehdi0095** [comment]
Hello! Do you have the MKS DLC32 MAX instruction? I need to turn on the Z axis so that first Z goes to the end sensor and then XY

**Comment by FabioRubino1** [comment]
Salve qualcuno sa perché sul volantino non c'è la possibilità di muovere l'asse z?
